extensions/amp-sticky-ad/0.1/amp-sticky-ad.js

```diff
@@ -77,6 +81,14 @@ class AmpStickyAd extends AMP.BaseElement {
     this.removeOnScrollListener_();
   }
 
+  /** @override */
+  collapsedCallback() {
+    this./*OK*/collapse();
```

Comment:
Sorry for waffling on this, but I now think it's more clear to call `toggle(false)`. 😬